src/cmap/commands.ts

```diff
@@ -30,7 +30,7 @@ export type WriteProtocolMessageType = Query | Msg | GetMore | KillCursor;
 
 /** @internal */
 export interface OpQueryOptions extends CommandOptions {
-  socketTimeout?: number;
+  socketTimeoutMS?: number;
```

Comment:
We had an inconsistency where `socketTimeoutMS` was being ignored because we only checked for `socketTimeout`. You'll see that update in a number of places.